### PR TITLE
tests

### PR DESCRIPTION
**Pull request by scheshire**
added tests and test data

### CODE REVIEW

File: sentiment-analysis/pom.xml [hunk 1]
```diff
@@ -21,6 +21,12 @@
       <artifactId>stanford-parser</artifactId>
       <version>3.9.1</version>
     </dependency>
+    <dependency>
+	    <groupId>pl.pragmatists</groupId>
```
Comment (michaelsuzukisagi) [inline sentiment-analysis/pom.xml]: 4 spaces instead of tabs.

File: sentiment-analysis/src/test/java/StanfordSentimentTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import static org.junit.Assert.*;
```
Comment (michaelsuzukisagi) [inline sentiment-analysis/src/test/java/StanfordSentimentTest.java]: Organise imports and not use star

File: sentiment-analysis/src/test/java/StanfordSentimentTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+import junitparams.FileParameters;
+import junitparams.JUnitParamsRunner;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(JUnitParamsRunner.class)
+public class StanfordSentimentTest {
```
Comment (michaelsuzukisagi) [inline sentiment-analysis/src/test/java/StanfordSentimentTest.java:14]: Formatting

File: sentiment-analysis/src/test/java/StanfordSentimentTest.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	@FileParameters("src/test/resources/testdata.csv")
+	public void test(int sentiment, String data) {
```
Comment (michaelsuzukisagi) [inline sentiment-analysis/src/test/java/StanfordSentimentTest.java:28]: Lets create a simpler test with correct asserstion

File: sentiment-analysis/src/test/java/StanfordSentimentTest.java (hunk 3 above)
Comment (scheshire) [reply sentiment-analysis/src/test/java/StanfordSentimentTest.java:14]: @michaelsuzukisagi format how?

File: sentiment-analysis/src/test/java/StanfordSentimentTest.java (hunk 4 above)
Comment (scheshire) [reply sentiment-analysis/src/test/java/StanfordSentimentTest.java:28]: is this better?

**Pull request merged**